lib/origin_simulator/recipe.ex

```diff
@@ -12,7 +12,8 @@ defmodule OriginSimulator.Recipe do
   # TODO: parameters don't make sense, need fixing
   @spec parse({:ok, binary(), any()}) :: binary()
   def parse({:ok, "[" <> body, _conn}), do: Poison.decode!("[" <> body, as: [%__MODULE__{}])
-  def parse({:ok, body, _conn}), do: Poison.decode!(body, as: %__MODULE__{})
+  def parse({:ok, body, _conn}), do: String.replace(body, "\n", "") |> Poison.decode!(as: %__MODULE__{})
```

Comment:
`String.replace` allows new lines in the JSON. It could go when the placeholder logic will be implemented. Or stay as it doesn't do any harm.